src/styles/main.scss

```diff
@@ -1,3 +1,75 @@
+$day-size: 100px;
+$day-gap: 0.01 * $day-size;
+$calendar-padding: 0.1 * $day-size;
+$calendar-width: $day-size * 7 + $day-gap * 6 + 2 * $calendar-padding;
+$day-count: 29;
```

Comment:
just change the highest number of days from 29 to 31 in $day-count variable
cause the bud is still present